src/features/suggestProduct/SuggestProduct.tsx

```diff
@@ -0,0 +1,126 @@
+import {useFormik} from 'formik';
+import * as Yup from 'yup';
+import {useQuery} from "@tanstack/react-query";
+import {getAllCategories} from "../../services/api.ts";
+import {useNavigate} from "react-router-dom";
+
+interface ProductSuggestion {
+    title: string;
+    price: number;
```

Comment:
very cool! just a suggestion - asking a customer for a price as a suggestion for a product might not be ideal. Perhaps you can use price if someone wants to upload a product through your platform :D

Reply:
Nice catch, that's very true actually. I think I'm going to leave it for now just for the purpose of the project to showcase a number input in the form. But definitely if this was a real-world application, that would be the way to go 😄